src/styles/main.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "../styles/block/calendar.scss";
+@import "../styles/block/days.scss";
+@import "../styles/block/month.scss";
+@import "../styles/block/reset.scss";
+
+@import "../styles/utilis/mixin.scss";
+@import "../styles/utilis/variables.scss";
```

Comment:
these should be the top imports, specially variables, and they are missing '_' 
and mixin is an empty file